src/jiraUpdateIssue.gs

```diff
@@ -103,36 +103,28 @@ function updateJiraIssues(headerRow, dataRows) {
  * @param value - the value from the spreadsheet for the specified field
  */
 function formatFieldValueForJira(fieldDefinition, value) {
-  if (fieldDefinition.key == "labels") {
```

Comment:
this was the bug I was relying on the key always being "label" when users can create a custom "label" field